Added Deep.BI - Predictive Analytics & AI Platform (#71)
apache/nifi-site: source/powered-by-nifi.md

# Powered by Apache {{< project-label >}}

## [Deep.BI](https://www.deep.bi)

> Deep.BI helps enterprises leverage next-gen data & AI pipelines to process hundreds of thousands of real-time events per second, powered by Apache Druid & Apache Flink. Our platform and solutions leverage NiFi as a core component in building robust cybersecurity solutions for critical applications and as a core component of our data processing architecture.

Reviewer: ```suggestion
Deep.BI helps enterprises leverage next generation data and AI pipelines to process hundreds of thousands of real-time events per second, powered by Apache Druid and Apache Flink. The Deep.BI data processing platform leverages Apache NiFi to build robust cybersecurity solutions for critical applications.
```

Author: Hi @exceptionfactory, thank you very much for your reply and suggestions, I think they work really well! I'll confirm it with the team to be fully sure.

I totally understand your concern, I probably should've asked one of our team members to do it from their active GitHub accounts.

I'll send an email to officially confirm this from our company email domain, I think that'll be the easiest, thank you!

Reviewer: Thanks for the reply @jmalcone546!

Sending an email from your company domain sounds good. I will plan on merging the addition once I receive the email confirmation.
